README.md

```diff
@@ -118,7 +119,33 @@ To configure your documentation to use exapandable sidebar headings like the exa
 
 *Tip: if you want to add all files in a sub-directory to your expandable dropdown section you can use the `:glob:` directive instead of listing out each page (see example above for the How-to Guides, API Reference and Explanations sections). This will list out each page in alphabetical order, so if you want a specific order you will need to list the pages out individually in the `toctree` (see example above for the Tutorials section)*
 
+## Enable translations
+
+First, coordinate with the Translations team at https://github.com/qiskit-community/qiskit-translations to express your interest and to coordinate setting up the infrastructure.
+
+Once the Translations team is ready, then update your `conf.py`:
+
+* Ensure that `qiskit_sphinx_theme` is in the `extensions` setting.
+* Set the option `translations_list` to a list of pairs of the locale code and the language name, e.g. `[..., ("de_DE", "German")]`.
+* Set the option `content_prefix` to your project's URL prefix, like `ecosystem/finance`.
```

Comment:
There are a couple of things I don't like about this option, including the name not being clear what it is: https://github.com/Qiskit/qiskit_sphinx_theme/issues/303. 

I plan to make some improvements as part of Sphinx Theme 0.12. But I want to do that in a follow up PR to make the code review easier. 

For now, this simply copies the name and logic we had in all of our Terra and Ecosystem projects.

As long as we make any changes before 0.12 is released, it is safe for us to make breaking changes like renaming this.